internal/cmd/cmd.go

```diff
@@ -78,21 +82,23 @@ func GenManpages(args []string, stdin io.Reader, stdout io.Writer, stderr io.Wri
 	})
 }
 
-func runRootCommandOutput(args []string, stdin io.Reader, stdout io.Writer, stderr io.Writer, f func(*cobra.Command, io.Writer) error) int {
-	return runRootCommand(args, stdin, stdout, stderr, func(cmd *cobra.Command) error { return f(cmd, stdout) })
+// develMode turns on sub-commands and potentially flags that we do not expose during the build of the prototool binary
```

Comment:
nit: missing period

Reply:
I don't put periods at the end of short comments, this is all over the codebase lol, this is just instructive

Reply:
Ok.